packages/components/src/pages/SingleProject.vue

```diff
@@ -0,0 +1,107 @@
+<template>
+  <v-multi-page ref="page" :disabled-pages="disabledPages">
+    <v-project-picker
+      id="project-picker"
+      :projects="projects"
+      :message-success="messageCopiedCommand"
+    />
+    <v-record-app-maps
+      id="record-appmaps"
+      :editor="editor"
+      :project="selectedProject"
+      :complete="hasRecorded"
+    />
+    <v-open-app-maps id="open-appmaps" :app-maps="appMaps" />
+    <v-investigate-findings
+      id="investigate-findings"
+      :scanned="hasFindings"
+      :num-findings="numFindings"
+      :project-path="path"
+    />
+  </v-multi-page>
+</template>
```

Comment:
Do we need this file? We should try to limit ourselves to a single entrypoint for the install guide (`InstallGuide.vue`).